connector/src/main/scala/com/microsoft/kusto/spark/datasource/KustoOptions.scala

```diff
@@ -1,21 +1,20 @@
 package com.microsoft.kusto.spark.datasource
```

Comment:
Can you please create a new package called common, and put it there?
It is no longer relevant to datasource